scripts/commands/update/update.js

```diff
@@ -19,22 +20,41 @@ const { ColorToken, GradientToken, SolidToken } = require("../../models");
 const update = () => {
   // Next tokens are mapped onto legacy tokens. These must be normalized to
   // match the existing local tokens.
+  // const next = {
```

Comment:
@InteractiveTimmy -- I was wondering how you intended to handle all 3 formats -- standard, automated, and design tokens in the same merge flow, or if the next should just be the style dictionary format